config/locales/en.yml

```diff
@@ -218,6 +218,12 @@ en:
   authorize_follow:
     error: Unfortunately, there was an error looking up the remote account
     follow: Follow
+    following_html: 'Success! You (<strong>%{self}</strong>) are now following:'
```

Comment:
I was trying to keep the two pages looking similar in style, but now that I think about it, you're probably right.